src/FSharp.Data.GraphQL.Shared/Helpers/ObjAndStructConversions.fs

```diff
@@ -24,7 +24,8 @@ module internal ValueTuple =
     let fstv struct (a, _) = a
     let sndv struct (_, b) = b
 
-module internal Seq =
+[<AutoOpen>]
+module Seq =
```

Comment:
I suppose we need to open internals to that project instead of opening the module to public